drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -305,6 +305,13 @@ static int stm32_clock_control_init(struct device *dev)
 #ifdef CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL
 	LL_UTILS_PLLInitTypeDef s_PLLInitStruct;
 
+#ifdef CONFIG_SOC_SERIES_STM32G4X
+	/* set power boost mode */
```

Comment:
Let's move this at the very end of the function.

Reply:
This voltage regulator must be set to enable range1 Boost mode **before** enabling the PLL.
So in any case, before the `LL_PLL_ConfigSystemClock_HSI()` function in the stm32_clock_control_init